src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -451,6 +486,36 @@ const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, o
             onSubmit={onSubmitForm}
             options={sources}
           />
+
+          {configuration.featureFlags.assignSignalToEmployee && (
+            <FilterGroup data-testid="filterAssignedUserEmail">
+              <Label htmlFor="filter_assigned_user_email" isGroupHeader>
+                Toegewezen aan
+              </Label>
+              <div>
+                <Checkbox
+                  data-testid="filterNotAssigned"
+                  checked={state.options.assigned_user_email === 'null'}
+                  id="filter_not_assigned"
+                  name="notAssigned"
+                  onClick={onNotAssignedChange}
+                />
+                <AscLabel htmlFor="filter_not_assigned" label="Niet toegewezen" />
```

Comment:
I would prefer not to use `Asc<Component>` throughout the code but define it as a variant of the own `components/Label` component

Reply:
It is already used in this component. Besides that, if I use the `components/Label` component the alignment between the label and the checkbox is off.